pkg/rtc/participant_sdp.go

```diff
@@ -360,6 +375,9 @@ func (p *ParticipantImpl) setCodecPreferencesForPublisherMedia(
 // configure publisher answer for audio track's dtx and stereo settings
 func (p *ParticipantImpl) configurePublisherAnswer(answer webrtc.SessionDescription) webrtc.SessionDescription {
 	offer := p.TransportManager.LastPublisherOffer()
+	if offer == nil {
```

Comment:
is it possible a nil offer when setting answer?

Reply:
No, it should not be. Left over from when I was trying to do the subscriber peer connection as single peer connection I think. Will remove.